core/src/main/java/io/questdb/cairo/wal/WalWriterEvents.java

```diff
@@ -266,6 +266,11 @@ void rollback() {
         // Not truncating the files saves from reading complexity.
     }
 
+    void sync(boolean async) {
+        eventMem.sync(async);
```

Comment:
nit: we should always sync metadata files, so the argument is redundant.

Reply:
fixed here https://github.com/questdb/questdb/pull/3304/commits/e2a1fce67513ec949fd420febeaa102c8391d169